src/pages/home/report/ReportAttachments.tsx

```diff
@@ -55,7 +56,8 @@ function ReportAttachments({route}: ReportAttachmentsProps) {
                 ComposerFocusManager.setReadyToFocus();
             }}
             onCarouselAttachmentChange={onCarouselAttachmentChange}
-            shouldShowNotFoundPage={!isLoadingApp && !report?.reportID}
+            shouldShowNotFoundPage={!isLoadingApp && type !== CONST.ATTACHMENT_TYPE.SEARCH && !report?.reportID}
+            isAuthTokenRequired={type === CONST.ATTACHMENT_TYPE.SEARCH}
```

Comment:
We need to get AuthTokenRequired logic based on the attachment because we also have public resources. Always attaching the auth token could cause [this issue](https://github.com/Expensify/App/issues/50006).